### PR TITLE
Feature Descriptor

### PR DESCRIPTION
**Pull request by dstrukt**
TODO: 
- [x] Add "X" to the accordion section
- [x] Improve the (?) asset (to provide .svg asset)
- [x] Make (?) default color `--neutral-500` and `height: 1.5rem` & `width: 1.5rem`
- [x] Add :hover state to the ( ? ) button
- [x] Refine copy

@dennisreimann may need some help here + anything else you want to explore ... whatever we do here I can copy to the other views.

First pass to get the ball rolling.

Current progress:
output

I will be polishing the design a bit more, but @pavlenex we should decide what we want to say for:

WRITE:
- [x] Invoices
- [x] Payment Requests
- [x] Pull Payments
- [x] Payouts

If we're feeling like 1-2 sentences is too much text, we could also do something like this as well:
image

One thing that would be interesting to explore would be opening this accordion for first time users / store creation, but dealing with state could be difficult or unreliable. Open to thoughts here.

closes: #3266

### CODE REVIEW

**Comment by dstrukt** [comment]
Update.

Added the todo items to the OP.

Screen Shot 2022-06-09 at 3 10 29 PM

**Comment by pavlenex** [comment]
> If we're feeling like 1-2 sentences is too much text, we could also do something like this as well:

Agreed should be explained in a few words. And we can always provide a tooltip for people to learn more.

 Invoices - have expiration
  Payment Requests - don't expire
  Pull Payments - no idea what to write
  Payouts - no idea, withdrawals or something

**Comment by dstrukt** [comment]
Hmm?

It's there, the changes are currently in the Invoice List view file, just has Payment Request copy as placeholder while I work on the UI & until we decided on something better.

**Comment by dstrukt** [comment]
@pavlenex I agree it should be concise as possible, but imo that might be too much. Will likely be tweaking a bit, but an attempt to be concise, but still give functional context:

### Payment Requests:

Payment Requests allow store owners to create long-lived invoices. 
Funds paid to a payment request use the exchange rate at the time of payment.

### Invoices:

An invoice is a document issued by the seller to a buyer to collect payment.
An invoice must be paid within a defined time interval at a fixed exchange rate.
Invoices have expiration because they lock the exchange rate within a specified time frame to protect the receiver from price fluctuations.

### Pull Payments:

Pull Payments allow the receiver to submit a request to pull payments from your wallet at a time the receiver sees fit.
You can then approve and release funds to the receiver at your convenience.

### Payouts:

This feature allows you to process pull payments, in the form of refunds, salary payouts, or withdrawals.

**Comment by pavlenex** [comment]
@dstrukt By tooltip I was referring to learn more type of thing, which you already added. So providing a very concise description of a feature in a sentence or two tops, then giving ability to go learn more if they need to is what I am proposing. 😄

**Comment by dstrukt** [comment]
Ooo I see, apologies, just wanted to clarify / make sure we were on the same page (which sounds like we are)!

**Comment by dennisreimann** [comment]
I've rebased this PR and updated the info icons. My impression is, that the icon is a bit too small, so that we need to bump its size, which doesn't work well in the normal text flow, where we have a font-size of 1rem and the icon now has 1.25rem. 

Can we adjust this, @dstrukt?

**Comment by dstrukt** [comment]
Yes! I left it as I didn't want to override or patch the sizing without consulting you on the best way to do so .. feel free!

**Comment by dennisreimann** [comment]
Updated it.

**Comment by dstrukt** [comment]
Awesome, thank you @dennisreimann will keep pushing on this ticket this evening and try to get it ready to be reviewd.

**Comment by dennisreimann** [comment]
Rebased, because there was a conflicts after other PR had been merged to master.

File: BTCPayServer/wwwroot/main/site.css [hunk 1]
```diff
@@ -96,6 +96,8 @@ a.unobtrusive-link {
 svg.icon.icon-info {
     color: var(--btcpay-neutral-500);
     transition: 200ms;
+    width: 1.5rem;
+    height: 1.5rem;
```
Comment (dennisreimann) [inline BTCPayServer/wwwroot/main/site.css:100]: I think we should first of all have the proper size in the SVG and then we can adjust width and height via CSS. https://github.com/btcpayserver/btcpayserver/pull/3831/files#diff-d0d2f4f86f403e6111f1a616e2551ff942201f853750abb443fc8ab0c05e9978R7

**Comment by NicolasDorier** [comment]
still in draft?

**Comment by dstrukt** [comment]
Yeah, need to finish up the copy, will bring out of draft this evening hopefully!

**Comment by dstrukt** [comment]
Attempt to extract and refine some of the copy from the docs for this context. It could still be shorter or better written, but it's a decent first attempt, and this isn't my strong suit.

This was tough, but it definitely helped me think through some ways some of the functionality could be communicated better .. PRs & Issues to follow.

This is the copy currently, open to suggestions and improvements!

### Invoices
An invoice is a document issued by the seller to a buyer to collect payment.
Invoices must be paid within a defined time interval, at a fixed exchange rate.

### Payment Requests
A payment request is a persistent shareable page that enables the receiver to pay at their convenience.
Funds paid to a payment request use the exchange rate at the time of payment.
Requests can paid in partial payments, and the payment request will remain valid until it is paid in full or until a specified expiration time.

### Pull Payments
A Pull Payment enables the receiver to submit a claim to pull payments from your wallet at the their convenience.
You can then approve this claim and release the requested funds to the receiver.

### Payouts
Payouts enable you to process Pull Payment claims.
You can approve, sign, and release funds to the requestor, or reject and disregard the claim.

**Comment by dstrukt** [comment]
@pavlenex Tagging you again for a review, thoughts on direction for this one so we can get it closed out!

**Comment by dennisreimann** [comment]
Rebased it.

**Comment by NicolasDorier** [comment]
Ping @dennisreimann anything more needed to merge this? seems good to go

**Comment by dstrukt** [comment]
@NicolasDorier We need to refine copy and evaluate everything we're saying, cc @pavlenex @dennisreimann could be something to discuss in tomorrow's call if there's time!

**Comment by dennisreimann** [comment]
Yes, let's see what's needed still and then finalize it.

**Comment by dennisreimann** [comment]
Listing the explainers that need to be finalized:

- An invoice is a document issued by the seller to a buyer to collect payment. Invoices must be paid within a defined time interval, at a fixed exchange rate, to protect the receiver from price fluctuations.
- A payment request is a persistent shareable page that enables the receiver to pay at their convenience. Funds paid to a payment request use the exchange rate at the time of payment. Requests can paid in partial payments, and the payment request will remain valid until it is paid in full or until a specified expiration time.
- A Pull Payment enables the receiver to submit a claim to pull payments from your wallet at the their convenience. You can then approve this claim and release the requested funds to the receiver.

Each of those has a "Learn more" link to the particular docs at the end.

Feel free to provide input, @pavlenex @Bangalisch @Zaxounette — @dstrukt and I will take care of the rest then :)

**Comment by dstrukt** [comment]
To add, we also have the "invoice" as well.

cc @pavlenex @Bangalisch @Zaxounette

**Review by pavlenex — APPROVED** [review]
uTACK. Textual changes seem good to me, we can probably iron them out once we see what kind of questions (if any) people ask about this.

**Pull request merged**